SasquatchTV/SasquatchTV/EventFilter/MSEventFilter.h

```diff
@@ -9,7 +9,7 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  * Event filtering service.
  */
-@interface MSEventFilter : MSServiceAbstract
+@interface MSEventFilter : MSACServiceAbstract
```

Comment:
MSACEventFilter?

Reply:
It's introduced inside the app, do we need to rename it?

Reply:
It's not required, but if it's already contains "MS" prefix..